.github/renovate.json

```diff
@@ -3,7 +3,8 @@
     "github>rancher/renovate-config#release"
   ],
   "baseBranches": [
-    "main"
+    "main",
+     "/^release\\/.*/"
```

Comment:
It's OK for now, but we will have to change this once one of our release branches goes EOL.

Reply:
done